compose/gpu-support.md

```diff
@@ -6,7 +6,9 @@ title: Enabling GPU access with Compose
 
 Compose services can define GPU device reservations if the Docker host contains such devices and the Docker Daemon is set accordingly. For this, make sure to install the [prerequisites](../config/containers/resource_constraints.md#gpu) if you have not already done so.
 
-The examples in the following sections focus specifically on providing service containers access to GPU devices with Docker Compose. You can use either `docker-compose` or `docker compose` commands.
+The examples in the following sections focus specifically on providing service containers access to GPU devices with Docker Compose. 
+You can use either `docker-compose` or `docker compose` commands.  
+See also, [Compose command compatibility with docker-compose](cli-command-compatibility.md).
```

Comment:
Also missing for that file:
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L63
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L105
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L164

Reply:
Replaced.